clients/include/testing_gels_strided_batched.hpp

```diff
@@ -36,6 +37,272 @@ inline void testname_gels_strided_batched(const Arguments& arg, std::string& nam
     hipblasGelsStridedBatchedModel{}.test_name(arg, name);
 }
 
+template <typename T>
+inline hipblasStatus_t testing_gels_strided_batched_bad_arg(const Arguments& arg)
+{
+    auto hipblasGelsStridedBatchedFn
+        = arg.fortran ? hipblasGelsStridedBatched<T, true> : hipblasGelsStridedBatched<T, false>;
+
+    hipblasLocalHandle       handle(arg);
+    const int                M          = 100;
+    const int                N          = 101;
+    const int                nrhs       = 10;
+    const int                lda        = 102;
+    const int                ldb        = 103;
+    const int                batchCount = 2;
+    const hipblasOperation_t opN        = HIPBLAS_OP_N;
+    const hipblasOperation_t opBad      = is_complex<T> ? HIPBLAS_OP_T : HIPBLAS_OP_C;
+
+    const hipblasStride strideA = size_t(lda) * N;
+    const hipblasStride strideB = size_t(ldb) * nrhs;
+    const int           A_size  = strideA * batchCount;
+    const int           B_size  = strideB * batchCount;
```

Comment:
These should be hipblasStride, size_t, or ssize_t even though not required by your constants you want to use correct sized variables.